lib/Db/AssignedLabelsMapper.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * @copyright Copyright (c) 2020 Jakob Röhrl <jakob.roehrl@web.de>
+ *
+ * @author Jakob Röhrl <jakob.roehrl@web.de>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+
+namespace OCA\Deck\Db;
+
+use OCP\AppFramework\Db\Entity;
+use OCP\IDBConnection;
+use OCP\IGroupManager;
+use OCP\IUserManager;
+
+class AssignedLabelsMapper extends DeckMapper  {
+	private $cardMapper;
+	private $userManager;
+	/**
+	 * @var IGroupManager
+	 */
+	private $groupManager;
+
+	public function __construct(IDBConnection $db, CardMapper $cardMapper, IUserManager $userManager, IGroupManager $groupManager) {
+		parent::__construct($db, 'deck_assigned_labels', AssignedLabels::class);
```

Comment:
I think the easiest would be to just reuse the Label class here, since it should already work for the assignment as well. So you could do that and drop the `AssignedLabels` class for now.